downstream/modules/builder/proc-customize-ee-image.adoc

```diff
@@ -0,0 +1,78 @@
+
+.Procedure
+. Log in to `quay.io` via Podman:
++
+----
+$ podman login -u="[username]" -p="[token/hash]" quay.io
+----
+. Verify that you have access to pull the base image that you want to use.
+.. For example, to pull a `ee-minimal` image, check that you have access to https://quay.io/repository/aap/ansible-automation-platform-20-ee-minimal-rhel8[this repository].
+. Configure your {Builder} files to specify any additional content to add to the new {ExecEnvShort} image which is based off of `ee-minimal`.
+.. For example, to add the https://galaxy.ansible.com/kubernetes/core[Kubernetes Core Collection from Galaxy] to the image, fill out the `requirements.yml` file as such:
++
+====
+----
+---
+collections:
+  - kubernetes.core
+----
+====
+.. For more information on definition files and their content, refer to <<assembly-definition-file-breakdown,to definition file breakdown section>>.
+. In the {ExecEnvShort} definition file, specify the filepath to the original `ee-minimal` container via the `EE_BASE_IMAGE` field. In doing so, your final `execution-environment.yml` file will look like the following:
++
+.A customized `execution-environment.yml` file
+[example]
+====
+----
+---
+version: 1
+
+build_arg_defaults:
+  EE_BASE_IMAGE: ‘quay.io/aap/ansible-automation-platform-20-ee-minimal-rhel8’
+
+dependencies:
+  galaxy: requirements.yml
+----
+====
++
+NOTE: Since this exampleuses the community version of `kubernetes.core` and not a certified collection from {HubName}, we do not need to create an `ansible.cfg` nor reference that in our definition file.
+. Build the new {ExecEnvShort} image using the following command:
++
+[subs=+quotes]
+----
+$ ansible-builder build -t quay.io/_[username]_/_new-ee_
+----
+where `[username]` specifies your username, and `new-ee` specifies the name of your new container image.
+.. Use the `podman images` command to confirm that your new container image is indeed in that list:
++
+.Output of a `podman images` command with the image `new-ee`
+====
+----
+REPOSITORY          TAG     IMAGE ID      CREATED        SIZE
+localhost/new-ee    latest  f5509587efbb  3 minutes ago  769 MB
+----
+====
+. Verify your newly-created {ExecEnvShort} image via Ansible Navigator
+. Push your image to the container registry {HubName}:
++
+----
+$ podman push quay.io/[username]/new-ee
+----
+. Pull your new image in your automation controller instance:
+.. Navigate to automation controller
+.. From the side-navigational bar, click *Administration* > *Execution Environments*.
+.. Click *Add*.
+.. From the "Pull" dropdown list, select your new container image.
++
+NOTE: if your instance of Automation Hub is password or token protected, ensure that you have the appropriate container registry credential set up.
```

Comment:
Automation Hub is sometimes capitalized and sometimes not; should it be capitalized everywhere?

Reply:
Automation hub should be lower cased. I must've a missed a few mentions of it. Good catch